src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceKind.kt

```diff
@@ -9,8 +9,8 @@ enum class LicenceKind(
   val creationEventType: () -> LicenceEventType,
   val copyEventType: () -> LicenceEventType,
   val submittedEventType: () -> LicenceEventType,
-  val activatedDomainEventType: () -> LicenceDomainEventType,
-  val inactivatedDomainEventType: () -> LicenceDomainEventType,
+  val activatedDomainEventType: () -> LicenceDomainEventType? = { null },
+  val inactivatedDomainEventType: () -> LicenceDomainEventType? = { null },
```

Comment:
As domain events not yet added for `HDC_VARIATION` had to make domain events nullable - is it better to just add them now to avoid temporary nullable changes?